app/controllers/late_policies_controller.rb

```diff
@@ -70,8 +70,7 @@ def create
         flash[:notice] = "The penalty policy was successfully created."
```

Comment:
Can you check at which line the error is? Do we see successfully creation message on the screen?

Reply:
@yashad Successful creation message can be seen by user and he can also see the policy in the dropdown menu as well as late_poicies list.

Suucessful Creation of policy

![successful_creation](https://user-images.githubusercontent.com/32436613/67168795-90f1ce00-f375-11e9-8bdd-3877234cafe1.png)

Policy in dropdown menu

![policy_in_dropdown](https://user-images.githubusercontent.com/32436613/67168797-9818dc00-f375-11e9-8efe-900ae025f030.png)